hazelcast/src/test/java/com/hazelcast/instance/impl/TestUtil.java

```diff
@@ -251,4 +253,11 @@ public static boolean isPortAvailable(int port) {
     public static String setSystemProperty(String key, String value) {
         return value == null ? System.clearProperty(key) : System.setProperty(key, value);
     }
+
+    public static byte[] byteBufferToBytes(ByteBuffer buffer) {
```

Comment:
Why do we move this to here, but not ClientTestUtil?

Reply:
I thought it is too general to be in a Client specific util file. Don't you agree. I think the real place of it should be BitsUtil or something like that but I could not find such a util file.

Reply:
I see. I though since this is only used in the client tests in conjunction with other methods you moved to ClientTestUtil, it makes sense to move this there as well. I am OK with this as well, so I leave the decision to you